shipment_planning_maxoptra/wizard/shipment_maxoptra_schedule_import.py

```diff
@@ -110,7 +111,7 @@ def action_import_schedule(self):
                     "is not implemented yet."
                 )
             )
-        self.update_shipment_planning(delivery_batch_pickings)
+        self.update_shipment_planning(delivery_batch_pickings | pick_batches)
```

Comment:
Union of large recordsets is usually costly, so it's probably more performant to browse a new recordset according to ids (or only pass the ids if we don't need the recordset in `update_shipment_planning` function.